src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -110,6 +111,69 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
                 actualOfficeConnectModel, expectedOfficeConnectModel);
     }
 
+    /**
+     * Executes the given {@code command}, confirms that <br>
+     * - the returned {@link CommandResult} matches {@code expectedCommandResult} <br>
```

Comment:
perhaps the <br> here could be removed? as the line break is already taken care of